src/context/WalletProvider/DemoWallet/DemoMenu.tsx

```diff
@@ -1,33 +1,9 @@
-import { EditIcon } from '@chakra-ui/icons'
-import { MenuDivider, MenuItem } from '@chakra-ui/react'
-import { useCallback } from 'react'
-import { useTranslate } from 'react-polyglot'
-import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
-import { useWallet } from 'hooks/useWallet/useWallet'
+import { ManageAccountsMenuItem } from 'components/Layout/Header/NavBar/ManageAccountsMenuItem'
 
-import { WalletActions } from '../actions'
-
-const editIcon = <EditIcon />
-
-export const DemoMenu = () => {
-  const translate = useTranslate()
-  const { dispatch: walletDispatch } = useWallet()
-  const isAccountManagementEnabled = useFeatureFlag('AccountManagement')
-
-  const handleManageAccountsMenuItemClick = useCallback(() => {
-    walletDispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
-  }, [walletDispatch])
```

Comment:
This was weird, why would we wan't to open the wallet modal when clicking on the manage accounts menu item

Reply:
@NeOMakinG that would be the reason why you spotted https://github.com/shapeshift/web/issues/7047 in the first place - the demo wallet isn't supposed to support account management, hence why we route to the wallet modal if the user wants to manage accounts (see original implementation in https://github.com/shapeshift/web/pull/6724/files#diff-90443cf259d584322bf43780fc6195133b34a0efdd42474386d5fa3f96e0d6e5R17-R19).

The issue you spotted isn't happening in develop, precisely because we guard against opening the account management modal for demo wallet, see develop vs. this diff:

https://jam.dev/c/a64a20f9-abb2-47ba-965a-4ae2257971f9

Reply:
Now whether or not we want to potentially discuss more with @shapeshift/product re: 
- possibly letting the users into Account Management in demo wallet, knowing they'd only be able to disable accounts, not add any
- or possibly hide the option altogether

Is a diff story, but as far as this diff goes, let's not mix concerns

Reply:
Reverted, will ping products on the demo wallet issue